Tests/XCUITests/FakespotTests.swift

```diff
@@ -0,0 +1,32 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+import XCTest
+
+class FakespotTests: BaseTestCase {
+    func testFakespotAvailable() {
+        navigator.openURL("https://www.amazon.com")
+        waitUntilPageLoad()
+
+        // Search for and open a shoe listing
+        let website = app.webViews["contentView"].firstMatch
+        website.textFields["Search Amazon"].tap()
+        website.textFields["Search Amazon"].typeText("Shoe")
+        website.buttons["Go"].tap()
+        waitUntilPageLoad()
+        website.images.firstMatch.tap()
+
+        // Tap shopping cart icon on Awesome bar
+        // Note: I can't find the label for the shopping cart icon.
+        // Workaround: Tap somewhere left of the "Share" button
+        waitForExistence(app.buttons["TabLocationView.shareButton"])
```

Comment:
Could we use the accessibility ids to look for view elements? The id for the button is `AccessibilityIdentifiers.Toolbar.shoppingButton`.

Reply:
The `app.debugDescription` does not show `AccessibilityIdentifiers.Toolbar.shoppingButton` in the awesome bar. Such an icon is displayed and can be tapped. Here is the debugDescription:
```
                        Other, 0x7ff510a1bb60, {{7.0, 53.0}, {376.0, 42.0}}
                          TextField, 0x7ff510a3e0d0, {{48.0, 63.0}, {215.0, 22.0}}, identifier: 'url', label: 'Address Bar', placeholderValue: 'Search or enter address', value: www.amazon.com/gp/...
                          Button, 0x7ff510a3a870, {{343.0, 54.0}, {40.0, 40.0}}, identifier: 'TabLocationView.reloadButton', label: 'Reload page'
                          Button, 0x7ff510a3ac20, {{7.0, 54.0}, {40.0, 40.0}}, identifier: 'TabLocationView.trackingProtectionButton', label: 'Secure connection'
                          Button, 0x7ff510a3afd0, {{303.0, 54.0}, {40.0, 40.0}}, identifier: 'TabLocationView.shareButton', label: 'Share this page'
  ```

Reply:
@thatswinnie has that icon change the ID, wondering why @clarmso cannot see it when printint the elements... I will give this a try later today

Reply:
I can't see it either... only Tracking Protection, Share and Reload button are available via print(app.debugDescription)
<img width="301" alt="Screenshot 2023-09-18 at 11 41 05" src="https://github.com/mozilla-mobile/firefox-ios/assets/1897507/c89dde9d-0f50-4c78-bcf4-86b447cd04ab">

Reply:
Relying on the coordinates to tap on the `shoppingButton` (via `tapAtPoint`) is flaky on my end. I would not merge this PR before `debugDescription` could see the icon. :sa

Reply:
Agreed... that coordinates are specific to a device's screen size.. better to have the ID...